README.md

```diff
@@ -21,7 +21,71 @@ This node is used by Rootstock PowPeg signatories to interact with the Bridge co
 ## Software installation help
 Disclaimer: this documentation will be specific for macOS operating system.
 
-### **Java JDK 1.8**
+### Quick Setup
+
+1. Install Java OpenJDK 1.8 from the binaries (if it is not installed yet)
```

Comment:
IMO, I feel like since there are so many ways to install Java 8, most users will just default to their preferred way to installing it.